hack/generator/pkg/astmodel/std_references.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+var (
+	// References to standard Go Libraries
+	FmtReference     PackageReference = MakeExternalPackageReference("fmt")
+	JsonReference    PackageReference = MakeExternalPackageReference("encoding/json")
+	ReflectReference PackageReference = MakeExternalPackageReference("reflect")
+	TestingReference PackageReference = MakeExternalPackageReference("testing")
+
+	// References to our Libraries
+	GenRuntimeReference PackageReference = MakeExternalPackageReference(genRuntimePathPrefix)
+
+	// References to other libraries
+	ApiExtensionsReference     = MakeExternalPackageReference("k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1")
+	ApiExtensionsJsonReference = MakeExternalPackageReference("k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/JSON")
```

Comment:
There are properties generated using types from that folder - maybe it's a subfolder of `ApiExtensionsReference` instead of a separate package. Definitely something I found during testing though.